tests/test_PR030_recordarray_in_numba.cpp

```diff
@@ -0,0 +1,37 @@
+// BSD 3-Clause License; see https://github.com/jpivarski/awkward-1.0/blob/master/LICENSE
+
+#include <iostream>
+#include <vector>
+
+#include "awkward/Slice.h"
+#include "awkward/fillable/FillableArray.h"
+#include "awkward/fillable/FillableOptions.h"
+
+namespace ak = awkward;
+
+int main(int, char**) 
+{
+  std::vector<std::vector<std::vector<double>>> vector =
+  {{{true, 1, 1.1}, {false, 2, 2.2}, {true, 3, 3.3}}, {}, {}, {}};
```

Comment:
This won't work: don't follow the original example this closely. It won't be easy to make C++ objects corresponding to the data (you'd have to define classes and all that), but you don't have to. Just call the appropriate `builder.XYZ` methods.